packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -188,30 +222,70 @@ export class VaAccordion {
   }
 
   render() {
-    const accordionItemIDs = [...this.el.children]
-      .filter((el) => el.tagName.toLowerCase() === 'va-accordion-item')
-      .map((el) => el.id);
-    return (
-      <Host>
-        {!this.openSingle && (
-          <button
-            ref={el => (this.expandCollapseBtn = el as HTMLButtonElement)}
-            onClick={() => this.expandCollapseAll(!this.expanded)}
-            aria-label={
-              this.expanded
-                ? i18next.t('collapse-all-aria-label')
-                : i18next.t('expand-all-aria-label')
+    const {uswds = false, openSingle} = this;
```

Comment:
I'm curious why `uswds` has to be set to false here.

Reply:
This was an artifact from development that I forgot to remove.